frontends/web/src/routes/settings/settings.tsx

```diff
@@ -288,6 +288,9 @@ class Settings extends Component<Props, State> {
                                                                 <div className={style.currency} key={`available-fiat-${index}`}>
                                                                     <div>
                                                                         <p className="m-none">{account.label}</p>
+                                                                        { account.label !== 'Bitcoin' ? (
```

Comment:
stringly typed comparison is very fragile in general. Use the `badges` field instead?

![image](https://user-images.githubusercontent.com/1225267/87004748-3841ba00-c1be-11ea-88a1-57c6f7153ca3.png)